packages/app/src/specs/SpecsListHoverCell.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div
+    ref="columnRef"
+    class="relative"
+  >
+    <slot
+      v-if="!shouldShowHover"
+      name="content"
+    />
+    <div
+      v-if="shouldShowHover"
+      ref="hoverButton"
+      class="inset-y-1 right-0 absolute"
+    >
+      <slot
+        name="hover"
+      />
+    </div>
+  </div>
+</template>
+
+/**
+  SpecsListHoverCell
+
+  Enables a cell in the SpecsList to have content switched out if the cell is hovered.
+
+  It contains two slots.  One for the main `content` and one for the `hover` content.
+
+  Note: This component contains styling specific for the SpecsList.
+ */
+<script setup lang="ts">
+import { ref, watch } from 'vue'
+import { useTimeoutFn, useElementHover } from '@vueuse/core'
+
+const props = defineProps<{
+  isHoverDisabled?: boolean
+}>()
+
+const columnRef = ref()
+const hoveredItem = ref()
+
+const shouldShowHover = ref(false)
+
+const isHoveredColumn = useElementHover(columnRef)
+const isHoveredItem = useElementHover(hoveredItem)
+
+let controls
+
+watch([isHoveredColumn, isHoveredItem], () => {
+  if (props.isHoverDisabled) {
+    return
+  }
+
+  if (controls) {
+    controls.stop()
+  }
+
+  if (isHoveredItem.value) {
+    isHoveredColumn.value = false
+  }
+
+  if (isHoveredColumn.value || isHoveredItem.value) {
+    if (shouldShowHover.value) return
+
+    controls = useTimeoutFn(() => {
+      shouldShowHover.value = true
+    }, 200)
+  } else if (!isHoveredColumn.value && !isHoveredItem.value) {
```

Comment:
Still thinking about this implementation, I wonder if we will have any perf/memory issues w/ constantly creating these hover event listeners.

Not sure if there is a better way, or if this even has any perf implications. I will think about it a little more. As an aside, why do we wait for 200 ms before showing the content?

Reply:
The delay was to prevent showing the buttons unless the user paused briefly over the cell.  This was intended to prevent the buttons from showing as a user quickly moved the mouse across the table.  

I share your concerns with perf/memory, but did some basic profiling and saw no major impacts.  If the specs list is changed from a virtual list that only shows a handful of things at a time to the full list in the DOM and only hides rows, then may be more of a concern.

I am currently reevaluating when these get shown based on your other related comments.